reports/tasks.py

```diff
@@ -46,27 +46,19 @@
         day_of_week=TRIAL_INSTANCES_REPORT_SCHEDULE_DAY_OF_WEEK
     )
 )
-def send_trial_instances_report(recipients=settings.TRIAL_INSTANCES_REPORT_RECIPIENTS):
+def send_trial_instances_report(
+        recipients=settings.TRIAL_INSTANCES_REPORT_RECIPIENTS):
     """
-    Generate and send a trial instance data report
+    Generate and send a new instance data report for the past month
```

Comment:
@spokerman12, nit: the new instances are still trial instances :)

Reply:
Understood. Changed like so.